index.html

```diff
@@ -96,112 +96,11 @@ <h3>LET'S CONNECT</h3>
         </a>
     </div>
 <hr class="bg-white fw-bolder">
-    <section class="container-fluid bg-white rounded-3" id="recentWorks">
+    <section class="container-fluid bg-white rounded-3">
         <h2 class="text-center p-5 text-decoration-underline fs-1">My recent works</h2>
```

Comment:
- [ **OPTIONAL** ] Good job moving the `See Live` and `See Source` buttons at the bottom of your popup as pointed out by the previous reviewer. But I think it would be better if you improve the design of your whole portfolio by sticking as much as possible to your choosing **[Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=ubZTS2MVguXE9Pc0-0)**. Each section and its content design should be close as much as possible to the ones from the Figma template. Eg.: Typography, Icons, fonts, images, spacing between and around different elements,...